gantry/core/utils.cpp

```diff
@@ -17,25 +17,41 @@ auto utils::get_node_id() -> can_ids::NodeId {
 }
 
 auto utils::register_config_by_axis(enum GantryAxisType which)
-    -> motor_driver_config::RegisterConfig {
+    -> tmc2130::TMC2130RegisterMap {
     switch (which) {
         case GantryAxisType::gantry_x:
-            return motor_driver_config::RegisterConfig{.gconf = 0x04,
-                                                       .ihold_irun = 0x71002,
-                                                       .chopconf = 0x40101D5,
-                                                       .thigh = 0xFFFFF,
-                                                       .coolconf = 0x60000};
+            return tmc2130::TMC2130RegisterMap{
+                .gconfig = {.en_pwm_mode = 1},
+                .ihold_irun = {.hold_current = 0x2,
+                               .run_current = 0x10,
+                               .hold_current_delay = 0x7},
+                .thigh = {.threshold = 0xFFFFF},
+                .chopconf = {.toff = 0b101,
+                             .hstrt = 0b101,
+                             .hend = 0b11,
+                             .tbl = 0b10,
+                             .mres = 0b100},
+                .coolconf = {.sgt = 0b110}};
 
         case GantryAxisType::gantry_y:
-            return motor_driver_config::RegisterConfig{.gconf = 0x04,
-                                                       .ihold_irun = 0x71002,
-                                                       .chopconf = 0x40101D5,
-                                                       .thigh = 0xFFFFF,
-                                                       .coolconf = 0x60000};
+            return tmc2130::TMC2130RegisterMap{
+                .gconfig = {.en_pwm_mode = 1},
+                .ihold_irun = {.hold_current = 0x2,
+                               .run_current = 0x10,
+                               .hold_current_delay = 0x7},
+                .tpowerdown = {},
+                .tcoolthrs = {.threshold = 0},
+                .thigh = {.threshold = 0xFFFFF},
+                .chopconf = {.toff = 0x5,
```

Comment:
Can we choose to either represent numbers for each register either in binary or hex? It's a bit confusing that chopconf here is in hex whereas above it's in binary.